README.md

```diff
@@ -64,316 +55,124 @@ machine during metric analysis.
+ingest_url = "https://ingest.YOUR_SIGNALFX_REALM.signalfx.com"; api_url = "https://api.YOUR_SIGNALFX_REALM.signalfx.com"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(". {$script} $(&{$args} @params)"))}
+```
 
-`& {Set-ExecutionPolicy Bypass -Scope Process -Force; $script = ((New-Object System.Net.WebClient).DownloadString('https://dl.signalfx.com/signalfx-agent.ps1')); $params = @{access_token = "YOUR_SIGNALFX_API_TOKEN"}; Invoke-Command -ScriptBlock ([scriptblock]::Create(". {$script} $(&{$args} @params)"))}`
+The agent will be installed as a Windows service and will log to the Windows Event Log.
 
-The agent should be installed to `\Program Files\SignalFx\SignalFxAgent`, and the default configuration file
-should be installed at `\ProgramData\SignalFxAgent`.
+### Step 2. Confirm your Installation 
 
-#### Chef
-We offer a Chef cookbook to install and configure the agent.  See [the cookbook
-source](./deployments/chef) and [on the Chef
-Supermarket](https://supermarket.chef.io/cookbooks/signalfx_agent).
+To confirm the SignalFx Smart Agent installation is functional on either platform, enter:
 
-#### Puppet
-We also offer a Puppet manifest to install and configure the agent on Linux.  See [the
-manifest source](./deployments/puppet) and [on the Puppet
-Forge](https://forge.puppet.com/signalfx/signalfx_agent/readme).
+```sh
+sudo signalfx-agent status
+````
 
-#### Ansible
-We also offer an Ansible Role to install and configure the Smart Agent on Linux.  See [the
-role source](https://github.com/signalfx/signalfx-agent/tree/master/deployments/ansible).
+The response you will see is:
 
-#### Salt
-We also offer a Salt Formula to install and configure the Smart Agent on Linux.  See [the
-formula source](https://github.com/signalfx/signalfx-agent/tree/master/deployments/salt).
+```sh
+SignalFx Agent version:           4.5.0
+Agent uptime:                     8m44s
+Observers active:                 host
+Active Monitors:                  16
+Configured Monitors:              33
+Discovered Endpoint Count:        6
+Bad Monitor Config:               None
+Global Dimensions:                {host: my-host-1}
+Datapoints sent (last minute):    1614
+Events Sent (last minute):        0
+Trace Spans Sent (last minute):   0
+````
+
+To verify the installation, you can run the following commands:
 
-#### Docker Image
-See [Docker Deployment](./deployments/docker) for more information.
+```sh
+signalfx-agent status config - show resolved config in use by agent
+signalfx-agent status endpoints - show discovered endpoints
+signalfx-agent status monitors - show active monitors
+signalfx-agent status all - show everything
+````
 
-#### Kubernetes
-See our [Kubernetes setup instructions](./docs/kubernetes-setup.md) and the
-documentation on [Monitoring
-Kubernetes](https://docs.signalfx.com/en/latest/integrations/kubernetes-quickstart.html)
-for more information.
+#### Troubleshoot any discrepancies in the Installation 
```

Comment:
This should be step 3, no? Install, confirm, troubleshoot

Reply:
no, we are assuming that all will go well and that troubleshooting is not necessary as a step in all cases.